src/lib/Components/ScrollableTabBar.tsx

```diff
@@ -109,7 +109,7 @@ export default class ScrollableTabBar extends React.Component<ScrollableTabBarPr
 
     const { width: screenWidth } = Dimensions.get("window")
     // If the scrollview doesn't need to scroll, don't try to scroll it!
-    if (this.scrollViewWidth < screenWidth) {
+    if (this.scrollViewWidth < screenWidth || !this.scrollView) {
```

Comment:
Let's move it to the start of the function. Nothing needs to be done when we don’t have this ref.